applications/dirgraph_visualization/dirgraph_main.py

```diff
@@ -0,0 +1,619 @@
+import pysvzerod
+import pandas as pd
+import matplotlib.pyplot as plt
+import networkx as nx
+import json
+import dash
+from dash import html, dcc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import os
+from dirgraph_utils import set_up_0d_network
+
+
+'''
+This file enables the visualization of 0D simulation results in a web app that displays the 0D network 
+as a directed graph. Users can interactively select nodes to view their parameters and simulation results.
+
+Simply provide the filepath for your simulation input JSON file and specify the directory where 
+you want to save the output directed graph.
+
+Created by Emilin Mathew (emilinm@stanford.edu) 
```

Comment:
We usually don't add this to files because git automatically keeps track of who added what (unironically called `git blame`)